pandas/guide/source/pandas_setup.rst

```diff
@@ -108,3 +131,5 @@ To check in which branch are you:
 
 To change to another branch:
     | ``git checkout <branch_name>``
```

Comment:
Same, and if we say this here, we should probably say it at the beginning, when cloning...